ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/external-lb/ipi-conf-vsphere-zones-multisubnets-external-lb-ref.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "ipi/conf/vsphere/zones/multisubnets/external-lb/ipi-conf-vsphere-zones-multisubnets-external-lb-ref.yaml",
```

Comment:
For this step, sounds not need to indicate any usage of external-lb, so it should be in `ipi/conf/vsphere/zones/multisubnets/`

Reply:
ok, got it.